__tests__/app.test.js

```diff
@@ -23,8 +23,41 @@ describe('General errors', () => {
     })
 });
 
+describe('GET /api', () => {
+    test('200: this endpoint returns a description of itself and an example response', () => {
+        return request(app)
```

Comment:
This test can be simplified and made dynamic by requiring the endpoints.json directly into the test file as you did in your model, and expecting the response.body to equal that object! 🧠

Reply:
Thank you, that's a much simpler and accurate test indeed. Made the change.